sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClient.cs

```diff
@@ -281,22 +285,27 @@ public virtual Response<T> GetDigitalTwin<T>(string digitalTwinId, CancellationT
             try
             {
                 // Serialize the digital twin object and write it to a stream
-                using MemoryStream memoryStream = await WriteToStreamAsync<T>(digitalTwin, _objectSerializer, cancellationToken).ConfigureAwait(false);
+                using MemoryStream memoryStream = await WriteToStreamAsync<T>(digitalTwin, _objectSerializer, cancellationToken)
+                    .ConfigureAwait(false);
 
                 // Get the response of the digital twin as a stream object
-                CreateOrReplaceDigitalTwinOptions options = ifNoneMatch != null ?
-                    new CreateOrReplaceDigitalTwinOptions { IfNoneMatch = ifNoneMatch.Value.ToString() } :
-                    null;
-                Response<Stream> digitalTwinStream = await _dtRestClient.AddAsync(digitalTwinId, memoryStream, options, cancellationToken).ConfigureAwait(false);
+                CreateOrReplaceDigitalTwinOptions options = ifNoneMatch == null
```

Comment:
1. Make condition easier to understand (use positive cases instead of negative cases)
1. `?` and `:` should be on the same line as the case they cover.